ra-optimisation/rao-commons/src/test/java/com/farao_community/farao/rao_commons/RaoUcteAliasesCreationTest.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.rao_commons;
+
+import com.powsybl.iidm.import_.Importers;
+import com.powsybl.iidm.network.Network;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+public class RaoUcteAliasesCreationTest {
+
+    @Test
+    public void createAliases() {
+        Network network = Importers.loadNetwork("TestCase12Nodes_withXnodes.uct", getClass().getResourceAsStream("/TestCase12Nodes_withXnodes.uct"));
+        RaoUcteAliasesCreation.createAliases(network);
+
+        assertSame(network.getBranch("DDE2AA   DDE3AA   1"), network.getBranch("DDE2AA   DDE3AA   ROT"));
```

Comment:
In the UCTE description, we have : 
` 1st character: UCTE country code   ///////
   2nd- 6th character:    short description of the geographical spot   /////////
   7th character: voltage level     //////////
   8th character: letter or figure for differentiating bus bars (optional)`

Shouldn't we be tolerant only on the 8th character of the node, and not on the last character of the node ?

Reply:
I think you're right... I tested with 12 nodes cases, and we did wrong inside it... the 8th caracter is always a white space this why I added the trim() method.
I'll fix that !